Sources/Auth/Services/Signer/EIP1271/RPCService.swift

```diff
@@ -0,0 +1,30 @@
+import Foundation
+import WalletConnectNetworking
+
+struct RPCService: HTTPService {
+    let data: Data
+    let projectId: String
+
+    var path: String {
+        return "/v1"
+    }
+
+    var method: HTTPMethod {
+        return .post
+    }
+
+    var scheme: String {
+        return "https"
+    }
+
+    var body: Data? {
+        return data
+    }
+
+    var queryParameters: [String : String]? {
+        return [
+            "chainId": "eip155:1",
```

Comment:
should we extend it for other eip155 chains?

Reply:
I think so. Updated, now chain ID getting from AuthRequestParams